pkg/operator/ceph/csi/spec.go

```diff
@@ -428,3 +433,51 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string) error {
 
 	return err
 }
+
+// ValidateCSIVersion checks if the configured ceph-csi image is supported
+func ValidateCSIVersion(clientset kubernetes.Interface, namespace, rookImage, serviceAccountName string) error {
+	var ownerRef metav1.OwnerReference
+	timeout := 15 * time.Minute
+
+	logger.Infof("detecting the ceph csi image version for image %s", CSIParam.CSIPluginImage)
+
+	pod, err := k8sutil.GetRunningPod(clientset)
+	if pod.GetOwnerReferences() == nil || len(pod.GetOwnerReferences()) == 0 {
+		return errors.Wrapf(err, "could not parse the owner of the rook operator")
+	}
```

Comment:
am suggesting something like below, let me know if anything wrong in  suggestion

```go
pod, err := k8sutil.GetRunningPod(clientset)
if err!=nil{
return errors.Wrapf(err, "could not parse the owner of the rook operator")
}
if pod == nil || len(pod.GetOwnerReferences()) == 0 {
		return errors.new("empty ownerreference in  rook operator pod")
}
```

Reply:
@shaas am not seeing code changes for above  suggestion

Reply:
Sorry! Forgot to push.